### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by axlecoffee**
### Changes
- Updated CONTRIBUTING.md to reflect the new rules (credit to thomas)

### CODE REVIEW

**Review by Ascynx — DISMISSED** [review]
Looks fine to me. (Not exactly sure All Rights reserved is a condition given by the Apache 2.0 license though)

**Comment by axlecoffee** [comment]
> Looks fine to me. (Not exactly sure All Rights reserved is a condition given by the Apache 2.0 license though)

we use it in the README....

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,4 +31,24 @@ Use [DevAuth](https://github.com/DJtheRedstoner/DevAuth). You do **not** need to
...
+- Any code change will have been seen (and approved) by at least 3 pairs of eyeballs (including the original writer) before it is able to be merged.
+- Contributor role is able to be given freely to (somewhat competent) developers without worrying about potential ramifications - reviews are still required, contributor just allows them to approve changes and merge once 2+ reviews have been given to the final PR.
+
+----------
+
+These are not advisories - GitHub will not allow you to merge a pull request without these being met. If you're confused about why you can't merge a PR, GitHub gives you a nice step-by-step breakdown of what you need to do at the bottom of the PR's page.
+
+---
+
+© 2023 SkyBlock-Central. All rights reserved.
```
Comment (TGWaffles) [inline CONTRIBUTING.md]:
```suggestion
© 2023 SkyBlock-Central. Subject to the Apache License 2.0.
```

**Review by TGWaffles — CHANGES_REQUESTED** [review]

**Comment by TGWaffles** [comment]
> > Looks fine to me. (Not exactly sure All Rights reserved is a condition given by the Apache 2.0 license though)
> 
> we use it in the README....

You added it to the README too. It doesn't actually make sense to say that with the license we have.

**Comment by Ascynx** [comment]
It's not because we weren't able to review your README changes that it means we approved them

**Review by Ascynx — APPROVED** [review]

**Review by TGWaffles — APPROVED** [review]
LGTM 👍

**Comment by axlecoffee** [comment]
WE FORGOT THE FUCKIN README

**Comment by Ascynx** [comment]
> WE FORGOT THE FUCKIN README

oh shit we did

**Comment by TGWaffles** [comment]
> WE FORGOT THE FUCKIN README

"we"

You can open a separate PR for the readme if you'd like.

**Pull request merged**